.env

```diff
@@ -14,6 +14,10 @@ REDIS_PORT=6379
 SUPERVISOR_HOST=backend
 SUPERVISOR_PORT=9001
 
+# Monitor Supervisor
+MON_SUPERVISOR_HOST=monitor
+MON_SUPERVISOR_PORT=9001
```

Comment:
btw is the 9001 port reserved for a special reason?

Reply:
its supervisors port